exportoptions/properties.go

```diff
@@ -17,16 +17,16 @@ const EmbedOnDemandResourcesAssetPacksInBundleDefault = true
 // ICloudContainerEnvironmentKey ...
 const ICloudContainerEnvironmentKey = "iCloudContainerEnvironment"
 
+// ICloudContainerEnvironment ...
+type ICloudContainerEnvironment string
```

Comment:
Why is this needed?

Reply:
This was moved from just a few lines down, to have the following consistent structure:
1, exportOptionKey
2, exportOptionValueType
3, exportOptionsValues